### PR TITLE
Contributing guidelines

### PR DESCRIPTION
**Pull request by vblagoje**
@ryanemerson please review

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Contributions from the community are essential in keeping Infinispan Management Console strong and successful.
+
+This guide focuses on how to contribute back to Infinispan Management Console using GitHub pull requests.
+If you need help with cloning, compiling or setting the project up in an IDE please refer to
+[this page](http://infinispan.org/docs/stable/contributing/contributing.html).
+
+## Legal
+
+All original contributions to Infinispan Management Console are licensed under the
```
Comment (tristantarrant) [inline CONTRIBUTING.md]: The Console must be ASL, not LGPL.

Comment (vblagoje) [reply CONTRIBUTING.md]: @tristantarrant see below the updated text

**Comment by vblagoje** [comment]
@ryanemerson hold this PR as we already have an encompassing PR for the entire project https://github.com/infinispan/infinispan/pull/5869

**Comment by vblagoje** [comment]
I think we should retire this one now that we have overall Infinispan contributing guidelines

**Comment by ryanemerson** [comment]
@vblagoje We should either copy `Contributing.MD` from `infinispan/infinispan` or link to that file in our readme.

**Comment by ryanemerson** [comment]
@vblagoje Are you going to revisit this PR or should we close?

**Pull request closed without merging**